server/kafka-consumer/src/main/java/org/hawkular/apm/server/kafka/KafkaZipkinSpansConsumer.java

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright 2015-2016 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.hawkular.apm.server.kafka;
+
+import java.util.List;
+import java.util.concurrent.Executor;
+
+import javax.annotation.PostConstruct;
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+import org.hawkular.apm.server.api.cdi.Eager;
+import org.hawkular.apm.server.api.services.SpanPublisher;
+import org.hawkular.apm.server.api.utils.zipkin.ZipkinSpanConvertor;
+import org.jboss.logging.Logger;
+
+import zipkin.Span;
+import zipkin.collector.kafka.KafkaCollector;
+import zipkin.storage.AsyncSpanConsumer;
+import zipkin.storage.AsyncSpanStore;
+import zipkin.storage.SpanStore;
+import zipkin.storage.StorageAdapters;
+import zipkin.storage.StorageComponent;
+
+
+/**
+ * @author Pavol Loffay
+ */
+@Eager
+@ApplicationScoped
+public class KafkaZipkinSpansConsumer {
+
+    private static final Logger log = Logger.getLogger(KafkaZipkinSpansConsumer.class);
+
+    private final String zookeeper;
+
+    @Inject
+    private SpanPublisher spanPublisher;
+
+
+    public KafkaZipkinSpansConsumer() {
+        this.zookeeper = System.getenv("KAFKA_ZOOKEEPER");
+    }
+
+    @PostConstruct
+    public void initializeZipkinKafkaCollector() {
+        if (zookeeper == null) {
+            return;
```

Comment:
Sorry didn't read the top comments before review - so now I understand why you did this :+1: